bin/tessel-2.js

```diff
@@ -277,6 +277,15 @@ parser.command('update')
   })
   .help('Update the Tessel firmware and openWRT image');
 
+parser.command('version')
+  .option('timeout', timeoutOption)
+  .option('name', nameOption)
+  .callback(function(opts) {
+    controller.tesselFirmwareVerion(opts)
+      .then(closeSuccessfulCommand, closeFailedCommand);
+  })
+  .help('Tessel firmware version');
```

Comment:
I recommend this text gets changed to "Display Tessel's current firmware version" to remain consistent with the other help commands which are generally more verbose.